src/Language/PureScript/CoreFn/Desugar.hs

```diff
@@ -209,6 +212,7 @@ findQualModules decls =
   fqValues :: A.Expr -> [ModuleName]
   fqValues (A.Var q) = getQual' q
   fqValues (A.Constructor q) = getQual' q
+  fqValues (A.TypeClassDictionaryConstructorApp C.IsSymbol _) = getQual' C.IsSymbol
```

Comment:
This lets us implicitly add an import of `Data.Symbol` if the `IsSymbol` class is used.  The import also gets deduped later if it's already there.
Is this a good place to add this?